src/scenes/Lines/scenes/Editor/index.tsx

```diff
@@ -28,30 +28,30 @@ import General from './General';
 import { setSavedChanges } from 'actions/editor';
 import { withRouter } from 'react-router-dom';
 import { selectIntl } from 'i18n';
-import { createSelector } from 'reselect';
 import messages from './messages';
-import { filterNetexOperators } from 'reducers/organisations';
-
-const selectFlexibleLine = createSelector(
-  state => state.flexibleLines,
-  (_, match) => match,
-  (flexibleLines, match) =>
-    (match.params.id && flexibleLines?.find(l => l.id === match.params.id)) ||
-    new FlexibleLine({
-      transportMode: VEHICLE_MODE.BUS,
-      transportSubmode: VEHICLE_SUBMODE.LOCAL_BUS
-    })
-);
-
-function useLoadDependencies(match, history) {
+import {
+  filterNetexOperators,
+  OrganisationState
+} from 'reducers/organisations';
+import { GlobalState } from 'reducers';
+import { RouteComponentProps } from 'react-router';
+import { MatchParams } from 'http/http';
+import { isBlank } from 'helpers/forms';
+import ServiceJourneysEditor from 'scenes/Lines/scenes/Editor/ServiceJourneys';
+import { FlexibleLinesState } from 'reducers/flexibleLines';
+
+const useLoadDependencies = ({
+  match,
+  history
+}: RouteComponentProps<MatchParams>) => {
   const [networksIsLoading, setNetworksIsLoading] = useState(true);
   const [flexibleLineIsLoading, setFlexibleLineIsLoading] = useState(true);
   const [
     flexibleStopPlacesIsLoading,
     setFlexibleStopPlacesIsLoading
   ] = useState(true);
 
-  const dispatch = useDispatch();
+  const dispatch = useDispatch<any>();
```

Comment:
Kan det inte vara av typen `GlobalState` 🙊 ?

Reply:
Nei, for det er en asynkron dispatch :/ Så krever litt annerledes type. Ikke helt sikker på hvordan man løser det sånn på stående fot